activation/wire/wire_v2.go

```diff
@@ -31,7 +31,8 @@ type ActivationTxV2 struct {
 	// A marriage is permanent and cannot be revoked or repeated.
 	// All new IDs that are married to this ID are added to the equivocation set
 	// that this ID belongs to.
-	Marriages []MarriageCertificate `scale:"max=256"`
+	// Note: 255 IDs + the owner of the ATX = 256 total IDs in a marriage.
```

Comment:
The owner should sign a certificate that they married "themselves" to make malfeasance proofs easier. For details see https://github.com/spacemeshos/go-spacemesh/pull/6043.